src/jit/hwintrinsicxarch.cpp

```diff
@@ -218,20 +218,26 @@ unsigned Compiler::simdSizeOfHWIntrinsic(NamedIntrinsic intrinsic, CORINFO_SIG_I
     return simdSize;
 }
 
+// TODO_XARCH-CQ - refactoring of numArgsOfHWIntrinsic fast path into inlinable
+// function and slow local static function may increase performance significantly
```

Comment:
I don't think we need this comment. As far as I know there is no evidence that this has a measurable impact on throughput.

Reply:
Thanks for review. May correct it in next PR?

Reply:
Yes, that's fine.